### DISCUSSION TITLE
Chime and Music Assistant - resume plaback skips to next track

### DISCUSSION THREAD

**Discussion by pgale** [nimroddolev/chime_tts#158] [Q&A]
Hi,

I use Music Assistant on snapcast client players on Android devices. I've just been experimenting with chime_tts for my announcements and like the features over the in-built functionality of MA and TTS handling (no custom chime sound possible).

However, when announcing, the player stops and the announcement plays but on resuming the playback, the player skips to the next track in the playlist. Not sure if this is due to the commands chime_tts sends or something not right with Music Assistant.

Any thoughts? Thanks.

**Comment by breakone9r** [comment]
I have the same issue, but in addition, even if it wasn't playing a track when I trigger an announcement, it'll start playing whatever's in the queue.

**Comment by Multimull** [comment]
Exact same behaviour here... Any updates or workarounds for this odd behaviour?
